ui/partials/perf/graphsctrl.html

```diff
@@ -79,7 +79,7 @@
           <i class="text-warning fa fa-exclamation-circle"></i>
           <a href="perf.html#/alerts?id={{tooltipContent.alertSummary.id}}">
             Alert #{{tooltipContent.alertSummary.id}}</a>
-          <span class="text-muted">- {{tooltipContent.alert.getStatusText()}}
+          <span class="text-muted">- {{tooltipContent.alert.status == 4 ? tooltipContent.alertSummary.getStatusText() : tooltipContent.alert.getStatusText()}}
```

Comment:
Could you change `tooltipContent.alert.status == 4` to `tooltipContent.alert.isAcknowledged()` ? That'll be a bit more clear I think.